parsec/serde/schema.py

```diff
@@ -90,26 +92,35 @@ def get_obj_type(self, obj):
     setting `type_field` class property.
     """
 
-    type_field = "type"
-    type_field_remove = True
-    type_schemas = {}
-    fallback_type_schema = None
-    _instantiated_schemas = None
-    _instantiated_fallback_schema = None
+    type_field: str = "type"
+    type_schemas: Dict[Enum, Schema] = {}
+    fallback_type_schema: Schema = None
+    _instantiated_schemas: Dict[Union[Enum, str], Schema] = None
+    _instantiated_fallback_schema: Dict[Union[Enum, str], Schema] = None
```

Comment:
bad typing: fallback is a single schema (so should be `_instantiated_fallback_schema: Schema`)

Reply:
fixed